core/templates/dev/head/pages/exploration_player/PlayerPositionServiceSpec.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+* @fileoverview Unit tests for the player position service.
+*/
+
+describe('Player position service', function() {
+  beforeEach(module('oppia'));
+
+  var pts, pps, sof;
+  beforeEach(inject(function($injector) {
+    pts = $injector.get('PlayerTranscriptService');
+    pps = $injector.get('PlayerPositionService');
+    scof = $injector.get('StateCardObjectFactory');
+  }));
+
+  it('should record answer submission correctly', function() {
+    pps.recordAnswerSubmission();
+    expect(pps.hasLearnerJustSubmittedAnAnswer()).toBe(true);
+  });
+
+  it('should make learner submitted as false when navigation button is clicked',
```

Comment:
`...should record answer submission by the learner as false...`

Reply:
changed